examples/clinical_pipeline/default_onto_project.json

```diff
@@ -0,0 +1,732 @@
+{
+  "name": "clinical_pipeline_base",
+  "project_type": "single_pack",
+  "ontology": {
+    "name": "base_ontology",
```

Comment:
This is a copy of the `base_ontology.json` [file](https://github.com/asyml/forte/blob/master/forte/ontology_specs/base_ontology.json), do we need to copy this here?